script.js

```diff
@@ -278,9 +360,9 @@ require(['jquery','core/str'], function ($, Str)
                     .parent().parent().find('h3.sectionname').text());
```

Comment:
This should probably be .closest('PARENT_SELECTOR').find(h3.sectionname').text());